packaging/helm-charts/helm3/strimzi-kafka-operator/values.yaml

```
@@ -78,6 +78,27 @@ podDisruptionBudget:
   maxUnavailable:
   unhealthyPodEvictionPolicy: IfHealthyBudget
 
+operatorNetworkPolicy:
+  # If enabled, this will generate a networkPolicy for the strimzi-operator.
+  # This flag DOES NOT control operator generated networkPolicies!
+  # This flag ONLY controls access for the strimzi-operator deployment.
+  enabled: false
+  # For ingress we permit access to the metrics endpoint
+  ingress:
+  - ports:
+    - protocol: TCP
+      port: http
+  # The operator must communicate with DNS, the Kubernetes API server,
+  # and all managed Kafka components.
+  #
+  # Kafka components use numerous, configurable ports and may exist in
+  # "arbitrary" namespaces. This makes a generic egress NetworkPolicy
+  # impractical to define. Thus we set `{}` an unrestrictred value.
```

The ports the operator connects to are not really configurable. But there are many of them, they differ per compoenent etc. So I think the idea is correct, but maybe the wording similar to the suggestion below is a bit more exact:

```suggestion
  # The operands use numerous ports that differ per component and may exist in
  # "arbitrary" namespaces. This makes a generic egress NetworkPolicy
  # impractical to define. Thus we set `{}` an unrestrictred value.
```